packages/typescript/mixin.core.js

```diff
@@ -1,17 +1,22 @@
 const { Mixin } = require('hops-mixin');
 
 class TypescriptMixin extends Mixin {
-  configureBuild(webpackConfig, { allLoaderConfigs, jsLoaderConfig }) {
-    const { loader, options, exclude } = jsLoaderConfig;
-    const babelLoader = { loader, options };
-    const typescriptLoader = { loader: require.resolve('ts-loader') };
+  configureBuild(webpackConfig, { jsLoaderConfig }) {
+    jsLoaderConfig.test.push(/\.tsx?$/);
+    jsLoaderConfig.exclude = /node_modules/;
```

Comment:
Do we really want to change that value here? Is it even required to do so?

Reply:
You're right, we actually don't want to change that value.